src/components/TimeCardPage/TimeSheet.js

```diff
@@ -49,33 +47,91 @@ export default function Page() {
         // Callback for date-range picker - does any pre-processing when grabbing a new date 
         setStartDate(startDate); 
         setEndDate(endDate); 
+        currentTimesheetsToDisplay(userTimesheets, start); 
         //TODO - remove once we are finished setting up API calls for  this 
         console.log("New date range has been selceted:\n\t %s \nto \n\t%s", start, end); 
     }
     
-    const [userTimesheets,setTimesheets] = useState([]); 
+    const [userTimesheets, setTimesheets] = useState([]); 
+    const [currentTimesheets, setCurrentTimesheets] = useState([]);
     const [selectedTimesheet, setTimesheet] = useState(); 
     const columns = defaultColumns 
 
     //Pulls user timesheets, marking first returned as the active one
     useEffect(() => {
         // Uncomment this if you want the default one loaded 
-        // setTimesheet(testingTimesheetResp)
+        //setTimesheet(testingTimesheetResp)
         apiClient.getUserTimesheets().then(timesheets => {
             setTimesheets(timesheets); 
-            //By Default just render / select the first timesheet for now  
-            if (timesheets.length > 0) {
-                setTimesheet(timesheets[0])
-            }  
-        });  
+            //By Default just render / select the first timesheet for now 
+            currentTimesheetsToDisplay(timesheets, startDate); 
+        });
     }, [])
 
     const processTimesheetChange = (timesheet) => {
         //Adding the time entry to the table 
         apiClient.addTimeEntry(timesheet); 
     }
 
+    const currentTimesheetsToDisplay = (timesheets, currentStartDate) => {
+        // refactor with moment
+        const currentTimesheets = timesheets.filter(sheet => new Date(sheet.StartDate * 1000).toDateString() === currentStartDate.toDateString());
```

Comment:
Can we potentially change this to use the .isEqual syntax using moment for a day comparison? Just think it would be better functionality than converting to the string format of it and then comparing like that. 

`moment.unix(<Time object>).isEqual(<Other moment object>, 'day')` would return whether it is the same day or not

Reply:
yup will change that right now